compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -898,6 +899,20 @@ class Definitions {
   final def isCompiletime_S(sym: Symbol)(implicit ctx: Context): Boolean =
     sym.name == tpnme.S && sym.owner == CompiletimePackageObject.moduleClass
 
+  final def isCompiletimeAppliedType(sym: Symbol)(implicit ctx: Context): Boolean = {
+    def isOpsPackageObjectAppliedType: Boolean =
+      sym.owner == CompiletimeOpsPackageObject.moduleClass && Set(
```

Comment:
I'll put that `Set` in a val to avoid recomputing it on every call

Reply:
Good call. Done in 5e287cda8b18dd9c90bff69eb782e23afb820af9